### DISCUSSION TITLE
Better in place Markdown editing; option to integrate Typora as editor or other?

### DISCUSSION THREAD

**Discussion by azeia** [standardnotes/forum#2629] [Suggestions]
I recently subscribed to be able to use Markdown editing.

Before/in parallel I was/am using the open source Joplin https://joplinapp.org solution (syncing through dropbox at this time) in combination with Typora standalone MD editor, because Joplin supports the use of editing a note in an external app like Typora (creating a temporary file until you uncheck editing externally for that note in Joplin). 
Upto now Typora https://typora.io is for me by far the best in place md editor, even though the new in place md editing in Joplin works also fine.

What I am missing in standard notes md editors is better in place editing, specifically if you are used to include tables or tasks in your notes. It is difficult to edit tables, as it has to be done in the source and tasks can't be checked off unless you actually go to the md source.

So, in place features as in Typora or in the newest in place editor withing Joplin would be highly appreciated.

**Comment by myreli** [comment]
While it's not a markdown editor, the Super editor offers a great editing experience and supports Markdown Shortcuts. [You might want to try it](https://www.youtube.com/watch?v=bkp6nrGqjUM&t=370s).

Sorry for the late reply, I'm currently triaging older issues and registering responses for those wondering 😅.
